src/orca-loader/Elements.res

```diff
@@ -67,7 +68,7 @@ let make = (
            <iframe
            id ="orca-payment-element-iframeRef-${localSelectorString}"
            name="orca-payment-element-iframeRef-${localSelectorString}"
-          src="${ApiEndpoint.sdkDomainUrl}/index.html?fullscreenType=${componentType}&publishableKey=${publishableKey}&clientSecret=${clientSecret}&sessionId=${sdkSessionId}"
+          src="${ApiEndpoint.sdkDomainUrl}/index.html?fullscreenType=${componentType}&publishableKey=${publishableKey}&clientSecret=${clientSecret}&sessionId=${sdkSessionId}&endpoint=${endpoint}"
```

Comment:
i dont think this is needed .. you are not using this inside the iframe

Reply:
![Uploading Screenshot 2024-05-08 at 10.46.47 PM.png…]()
Using this bro

Reply:
you are already getting the endpoint postmessage to the loadercontroller.res and also setting the value using setApiEndpoint() ..

use that value inside app.res and pass it as a prop

Reply:
if you are not getting the context then fine .. else do it like this